### PR TITLE
MINOR: Add Quanlong Huang to committers page

### PR DESCRIPTION
**Pull request by stiga-huang**
### What changes were proposed in this pull request?

This PR aims to add @stiga-huang to the committers page.

### Why are the changes needed?

To make the committers page up-to-date.

### How was this patch tested?

Manually review.

### CODE REVIEW

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM, @stiga-huang !

File: site/develop/committers.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Ferdinand Xu            | xuf          | Committer
 Yongzhi Chen            | ychena       | Committer
 Yiqun Zhang             | yqzhang      | Committer
 Aihua Xu                | zihuaxu      | Committer
+Quanlong Huang          | stigahuang   | Committer
```
Comment (dongjoon-hyun) [inline site/develop/committers.md:57]: Oh, the list is `alphabetic` order by ID.

Comment (dongjoon-hyun) [reply site/develop/committers.md:57]: Could you move your ID to a new location after `sseth`?

Comment (stiga-huang) [reply site/develop/committers.md:57]: Oops! Thanks for catching this!

**Comment by stiga-huang** [comment]
Thank you so much! @dongjoon-hyun

**Pull request merged**